lib/setupmanagers/qemuhck/network_manager.rb

```diff
@@ -159,15 +168,28 @@ def transfer_device_command(device_name, transfer_net, share_path, qemu_replacem
         net_base = "#{transfer_net}.0/24"
         smb_server = "#{transfer_net}.4"
 
-        netdev_options = "net=#{net_base},smb=#{path},smbserver=#{smb_server},restrict=on,ifname=@net_if_name@"
+        net_smb_private = replace_string_recursive('@workspace@/smb_@run_id@', qemu_replacement_list)
+        net_smb_private_shellescaped = net_smb_private.shellescape
+
+        # Mapping the current user to root will make smbd assume it can drop and
+        # re-add supplementary groups. However, re-adding supplementary groups
+        # do not work because they are typically not mapped in the current user
+        # namespace. Map the current user to nobody when running smbd so that
+        # smbd will run in the non-root mode.
+        net_smb_cmd = "unshare --map-user=nobody smbd -l #{net_smb_private_shellescaped} -s #{net_smb_private_shellescaped}/smb.conf"
+
+        netdev_options = ",net=#{net_base},guestfwd=:#{smb_server}:445-cmd:#{net_smb_cmd},restrict=on"
```

Comment:
Are you sure that `smb_server` IP is correct? When QEMU starts smb+slirp it assigns  `<IP>.4` by default.

Reply:
SMB uses 2 ports 445 and 139, better to share both. 
```SMB ports are generally port numbers 139 and 445. Port 139 is used by SMB dialects that communicate over NetBIOS.```

Reply:
We don't let activate QEMU built-in smb logic so it should be safe to the address.
NetBIOS is quite old and it is a common practice to disable it for security if I remember correctly. Probably it's not worth adding code for such an obsolete protocol.

Reply:
Ok, will test and update you